test/test.js

```diff
@@ -125,6 +135,23 @@ function parseOptions() {
         "--reftest, --unitTest, --fontTest, and --masterMode must not be specified together."
       );
     })
+    .check(argv => {
+      if (
+        +argv.unitTest + argv.fontTest + argv.integration + argv.xfaOnly <=
+        1
+      ) {
+        return true;
+      }
+      throw new Error(
+        "--unitTest, --fontTest, --integration, and -- xfaOnly must not be specified together."
```

Comment:
I think the space before `xfaOnly` needs to be removed here since `--xfaOnly` is the parameter?

Reply:
Whoops, thanks for spotting this!